examples/tests/app_loader/main.c

```diff
@@ -0,0 +1,298 @@
+// \file
```

Comment:
I think this app is complex enough that it deserves a README describing what it is doing